src/components/Exercise.tsx

```diff
@@ -77,6 +78,7 @@ export const Exercise = ({
         <ExerciseQuestion
           {...props}
           {...state}
+          ref={(el: HTMLDivElement) => questionsRef.current[questionNumber] = el}
```

Comment:
Should questionNumber here add the `i` index var like the questionNumber prop below? Once I made that change it seems to work on the assessments ui